### DISCUSSION TITLE
What is the support on iOS and Android ?

### DISCUSSION THREAD

**Discussion by arreumb** [deepgram/community#86] [General help] [answered]
?

**Comment by BekahHW** [comment]
@arreumb, can you be more specific about the type of support you're looking for?

**Comment by arreumb** [comment]
Sorry.  My question is: is there any way to do speech-to-text for a web app on android and ios ?  I wasn't able to find a way to stream the audio from the browser to a server and receive back the text. Can deepgram be helpful on this ?

**Reply by BekahHW** [reply]
Is this helpful: [https://blog.deepgram.com/flutter-speech-to-text-tutorial/](https://blog.deepgram.com/flutter-speech-to-text-tutorial/)

**Comment by BekahHW** [comment, marked as answer]
@arreumb I apologize. I misunderstood the question. 

If you check out [this repo](https://github.com/deepgram-devs/speechracer), we're doing something similar and it works on mobile. It's a game that you can play live at [speechracer.com](https://speechracer.com/) that matches the transcription to the text. So, the answer to your question is yes, you can create a web app that also transcribes and works on mobile.

**Reply by arreumb** [reply]
Thanks a lot for your answer.
